termii_test.go

```diff
@@ -73,7 +86,8 @@ func TestSendTokenSuccess(t *testing.T) {
 	os.Setenv("TERMII_URL", termiiService.URL)
 	fileToStruct(filepath.Join("testdata", "send_token_request.json"), &req)
 
-	c := termii.NewClient()
+	termiiClient, termiiHttpClient := NewClientConfig()
+	c := termii.NewClient(termiiClient, termiiHttpClient)
```

Comment:
Leave existing tests, simply add a new test using the client from newClientConfig function